spec/services/upload_service_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'aws-sdk-s3'
+require 'rails_helper'
+
+RSpec.describe "upload service", type: :service do
```

Comment:
You may want to add test cases for the above comments about multiple calls the the same method